content/upload/upload_1.md

```diff
@@ -0,0 +1,411 @@
+	PAGE_URLPORT NUMBER(38,0),
+	PAGE_URLPATH VARCHAR(3000),
+	PAGE_URLQUERY VARCHAR(6000),
+	PAGE_URLFRAGMENT VARCHAR(3000),
+	REFR_URLSCHEME VARCHAR(16),
+	REFR_URLHOST VARCHAR(255),
+	REFR_URLPORT NUMBER(38,0),
+	REFR_URLPATH VARCHAR(6000),
+	REFR_URLQUERY VARCHAR(6000),
+	REFR_URLFRAGMENT VARCHAR(3000),
+	REFR_MEDIUM VARCHAR(25),
+	REFR_SOURCE VARCHAR(50),
+	REFR_TERM VARCHAR(255),
+	MKT_MEDIUM VARCHAR(255),
+	MKT_SOURCE VARCHAR(255),
+	MKT_TERM VARCHAR(255),
+	MKT_CONTENT VARCHAR(500),
+	MKT_CAMPAIGN VARCHAR(255),
+	SE_CATEGORY VARCHAR(1000),
+	SE_ACTION VARCHAR(1000),
+	SE_LABEL VARCHAR(4096),
+	SE_PROPERTY VARCHAR(1000),
+	SE_VALUE FLOAT,
+	TR_ORDERID VARCHAR(255),
+	TR_AFFILIATION VARCHAR(255),
+	TR_TOTAL NUMBER(18,2),
+	TR_TAX NUMBER(18,2),
+	TR_SHIPPING NUMBER(18,2),
+	TR_CITY VARCHAR(255),
+	TR_STATE VARCHAR(255),
+	TR_COUNTRY VARCHAR(255),
+	TI_ORDERID VARCHAR(255),
+	TI_SKU VARCHAR(255),
+	TI_NAME VARCHAR(255),
+	TI_CATEGORY VARCHAR(255),
+	TI_PRICE NUMBER(18,2),
+	TI_QUANTITY NUMBER(38,0),
+	PP_XOFFSET_MIN NUMBER(38,0),
+	PP_XOFFSET_MAX NUMBER(38,0),
+	PP_YOFFSET_MIN NUMBER(38,0),
+	PP_YOFFSET_MAX NUMBER(38,0),
+	USERAGENT VARCHAR(1000),
+	BR_NAME VARCHAR(50),
+	BR_FAMILY VARCHAR(50),
+	BR_VERSION VARCHAR(50),
+	BR_TYPE VARCHAR(50),
+	BR_RENDERENGINE VARCHAR(50),
+	BR_LANG VARCHAR(255),
+	BR_FEATURES_PDF BOOLEAN,
+	BR_FEATURES_FLASH BOOLEAN,
+	BR_FEATURES_JAVA BOOLEAN,
+	BR_FEATURES_DIRECTOR BOOLEAN,
+	BR_FEATURES_QUICKTIME BOOLEAN,
+	BR_FEATURES_REALPLAYER BOOLEAN,
+	BR_FEATURES_WINDOWSMEDIA BOOLEAN,
+	BR_FEATURES_GEARS BOOLEAN,
+	BR_FEATURES_SILVERLIGHT BOOLEAN,
+	BR_COOKIES BOOLEAN,
+	BR_COLORDEPTH VARCHAR(12),
+	BR_VIEWWIDTH NUMBER(38,0),
+	BR_VIEWHEIGHT NUMBER(38,0),
+	OS_NAME VARCHAR(50),
+	OS_FAMILY VARCHAR(50),
+	OS_MANUFACTURER VARCHAR(50),
+	OS_TIMEZONE VARCHAR(255),
+	DVCE_TYPE VARCHAR(50),
+	DVCE_ISMOBILE BOOLEAN,
+	DVCE_SCREENWIDTH NUMBER(38,0),
+	DVCE_SCREENHEIGHT NUMBER(38,0),
+	DOC_CHARSET VARCHAR(128),
+	DOC_WIDTH NUMBER(38,0),
+	DOC_HEIGHT NUMBER(38,0),
+	TR_CURRENCY VARCHAR(3),
+	TR_TOTAL_BASE NUMBER(18,2),
+	TR_TAX_BASE NUMBER(18,2),
+	TR_SHIPPING_BASE NUMBER(18,2),
+	TI_CURRENCY VARCHAR(3),
+	TI_PRICE_BASE NUMBER(18,2),
+	BASE_CURRENCY VARCHAR(3),
+	GEO_TIMEZONE VARCHAR(64),
+	MKT_CLICKID VARCHAR(128),
+	MKT_NETWORK VARCHAR(64),
+	ETL_TAGS VARCHAR(500),
+	DVCE_SENT_TSTAMP TIMESTAMP_NTZ(9),
+	REFR_DOMAIN_USERID VARCHAR(128),
+	REFR_DVCE_TSTAMP TIMESTAMP_NTZ(9),
+	DOMAIN_SESSIONID VARCHAR(128),
+	DERIVED_TSTAMP TIMESTAMP_NTZ(9),
+	EVENT_VENDOR VARCHAR(1000),
+	EVENT_NAME VARCHAR(1000),
+	EVENT_FORMAT VARCHAR(128),
+	EVENT_VERSION VARCHAR(128),
+	EVENT_FINGERPRINT VARCHAR(128),
+	TRUE_TSTAMP TIMESTAMP_NTZ(9),
+	LOAD_TSTAMP TIMESTAMP_NTZ(9),
+	CONTEXTS_COM_SNOWPLOWANALYTICS_SNOWPLOW_WEB_PAGE_1 VARIANT,
+	constraint EVENT_ID_PK primary key (EVENT_ID)
+);
+```
+
+***
+
+#### **Step 3:** Load the data
+
+{{% notice info %}}
+Please note that the instructions are only valid when using the Snowflake Classic Console!
```

Comment:
I could not upload this via the Snowflake Console as the file size was over the limit, unfortunately, it must be reduced for this method to work. We were considering some time ago to see if [this](https://docs.snowflake.com/en/user-guide/data-sharing-product-offerings.html) could work for instance.

Reply:
Oh no, thanks for flagging this. We have removed/reduced a bunch of fields and got the file to 49MB!